Geometry.cs

```diff
@@ -144,14 +158,46 @@ class Drawing {
 
    public Bound2 Bound {
       get {
-         if (mBound.IsEmpty) mBound = new (Polys.Select (a => a.Bound));
+         if (mBound.IsEmpty) mBound = new (ConvexHull);
          return mBound;
       }
    }
    Bound2 mBound;
 
-   public Bound2 GetBound (Matrix2 xfm) 
-      => new Bound2 (Polys.SelectMany (a => a.Pts.Select (p => p * xfm)));
+   IReadOnlyList<Point2> ConvexHull { 
+      get {
+         if (!mConvexHull.Any ()) ComputeConvexHull (Polys.SelectMany (a => a.Pts));
+         return mConvexHull;
+      } 
+   }
+   List<Point2> mConvexHull = new ();
+
+   void ComputeConvexHull (IEnumerable<Point2> pts) {
+      Point2 mPtY = pts.Min ();
+      List<Point2> mCH = new ();

```

Comment:
Don't use an m prefix for a local variable.

Reply:
Now using the same mConvexHull list instead of temp list